### DISCUSSION TITLE
Inheriting Entry attributes without enabling Multisite?

### DISCUSSION THREAD

**Discussion by sbrow** [statamic/cms#9882] [Q&A]
I think it's really useful to have the ability to inherit properties from other Entries, but override them when needed.

This is possible when using the multi-site feature, but seems to be tightly restricted to that use case. I've tried getting around it on a single site install by setting `origin` manually on the entry and adding `localizable: true` to the blueprint fields, and although this *almost* works, it breaks when I try to use an `Entries` field to make `origin` dynamic.

Is there any way to inherit Entry attributes from other Entries on a single site install? If not, would the team be amenable to adding it in the future?

For context, I am using the [simple-commerce](https://statamic.com/addons/duncanmcclean/simple-commerce) addon, and attempting to add Subscription functionality on top. Each order has a billing address and a shipping address, and I'd like for each subscription to inherit those values from the order that created them, unless the subscriber updates the subscription, at which point the default must be overridden.

Thank you :)

**Comment by edalzell** [comment]
That's not possible right now, and sounds like a feature request for Simple Commerce to set the collection used for orders. Then you could make your own collection and SC would use it.

**Reply by sbrow** [reply]
> That's not possible right now

That's unfortunate to hear. I'd love to see it in Statamic 5+, as the ability to automagically inherit attributes from other Entries has (IMO) a lot of applications outside of multi-site and the use-case I outlined. Maybe down the line I could try to open a PR for this, but at the moment I'm under a bit of a time crunch.

Either way, thanks for the prompt response and awesome product!

**Comment by jasonvarga** [comment]
You could make a `subscription` computed value on the `orders` collection.

https://statamic.dev/computed-values

**Reply by sbrow** [reply]
Thank you! I was hoping origin would work, since it'd be a lot simpler, but at the moment I think I'm looking at adding a bunch of computed values :)
